features/karaf/esh-ext/src/main/feature/feature.xml

```diff
@@ -83,6 +83,11 @@
     <bundle>mvn:org.eclipse.smarthome.binding/org.eclipse.smarthome.binding.lirc/${project.version}</bundle>
   </feature>
 
+  <feature name="esh-binding-meteoblue" version="${project.version}">
+    <feature>esh-base</feature>
+    <bundle>mvn:org.eclipse.smarthome.binding/org.eclipse.smarthome.binding.meteoblue/${project.version}</bundle>
+  </feature>
+
```

Comment:
Please remove this blank line.

Reply:
This one I don't get. There's a blank line after every `<feature>` block in the file. Why should this one be different?

Reply:
Upps sorry, github only showed me a small portion of the file. You are right, sorry for the confusion.